devices/ace3dsplus.cpp

```diff
@@ -584,3 +586,5 @@ class Ace3DSPlus : Flashcart {
 
 Ace3DSPlus ace3DSplus;
 }
+
+#endif
```

Comment:
If the ifdef is moved, this line needs to be moved from here to within the curly brace. Again, similar change for r4igoldcc